utils/api/relationships.ts

```diff
@@ -1,16 +1,47 @@
 import useSWR, { SWRResponse } from 'swr';
 import axios from 'axios';
-
 import type { RelationshipData, ErrorAPI } from 'types';
 
 export const useRelationships = (
   id: number
 ): SWRResponse<RelationshipData, ErrorAPI> =>
   useSWR(`/api/residents/${id}/relationships`);
 
+interface addRelationshipFormData {
+  personId: number;
+  otherPersonId: number;
+  type: string;
+  additionalOptions?: string[] | boolean;
+  isMainCarer?: string;
```

Comment:
Should this be a boolean?

Looks like we're representing true as "Y" and false as "N". (if the backend returns information like this we should modify the `.toResponse()` to convert the string representation to an actual boolean.

https://github.com/LBHackney-IT/social-care-case-viewer-api/blob/master/SocialCareCaseViewerApi/V1/Factories/ResponseFactory.cs#L145

Reply:
Ah I noticed this is a post but I would still advise using boolean, if our database needs to use "Y" or "N" we can handle that when creating the Relationship object

Reply:
Yeah, this is what the service API currently sends when getting relationships and what it requests when creating a relationship. I think you're right, this should be a boolean but as I'm currently focusing on getting the functionality to work I won't address this in this PR. I'll add it to our Jira ticket for a thing to do, sorry!